src/pages/GenerateKeys/index.tsx

```diff
@@ -132,6 +185,44 @@ const _GenerateKeysPage = ({
           </div>
         </div>
       </Paper>
+      <Paper className="mt20">
+        <Heading level={2} size="small" color="blueMedium" className="mb20">
+          <FormattedMessage defaultMessage="Withdrawal address" />
+        </Heading>
+        <Text className="mb20">
+          <FormattedMessage
+            defaultMessage="You may choose to provide a withdrawal address with your initial
+            deposit to automatically enable reward payments and also the ability
+            to fully exit your funds anytime after the Shanghai/Capella upgrade.
+            This address should be to a regular Ethereum address and will be the
+            only address funds can be sent to from your new validator accounts."
+          />
+        </Text>
+        <Text className="mb20">
+          <FormattedMessage defaultMessage="Paste your chosen address here to include it in the copy/paste CLI command below:" />
+        </Text>
+        <AddressInputContainer className="mb40">
+          <AddressInput
+            onChange={handleAddressChange}
+            value={withdrawalAddress}
+            placeholder="0x... (no ENS)"
+            maxLength={42}
+          />
+          <AddressIndicator>{addressIndicatorEmoji}</AddressIndicator>
+        </AddressInputContainer>
+        <Alert variant="error">
+          {isValidWithdrawalAddress ? (
+            <FormattedMessage defaultMessage="Make sure you have control over this address as this cannot be changed." />
```

Comment:
Right, none of these are actually errors, but "error" is the variant name for the `Alert` component that styles it red. Using red here since this is a critical note, where messing up the address you provide will lead to loss of funds.

Reply:
Agreed. This is a critical message.